src/main/java/com/jcabi/github/Status.java

```diff
@@ -0,0 +1,22 @@
+package com.jcabi.github;
+
+import com.jcabi.aspects.Immutable;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Null;
+
+@Immutable
+public interface Status {
+
+    @NotNull(message = "state is never NULL")
+    StatusState state();
+
+    // TODO optional
+    String targetUrl();
```

Comment:
But these methods are used in `RtStatuses`. TODO message was about "perhaps add guava's Optional here". However testing this with the actual Github API proved that assumption wrong.

I removed those comments.